Packages/Core/Runtime/Events/AtomEvent.cs

```diff
@@ -33,19 +40,17 @@ public class AtomEvent<T> : AtomEventBase
         [Tooltip("The number of old values (between 0-10) being replayed when someone subscribes to this Event.")]
         private int _replayBufferSize = 1;
 
-        private Queue<T> _replayBuffer = new Queue<T>();
+        /// <summary>
+        /// Queue used as a replay buffer.
+        /// </summary>
+        private readonly Queue<T> _replayBuffer = new Queue<T>();
 
+        /// <summary>
+        /// This function is called when the scriptable object goes out of scope.
+        /// </summary>
```

Comment:
is it really possible to repeat the unity documentation?

https://docs.unity3d.com/ScriptReference/ScriptableObject.OnDisable.html

here again, this is not a public interface, (which SHOULD be described) but instead a private method, within the context of unity. basically every maintainer should know by hearth when OnDisabled is called

Reply:
I'm in the habit of not assuming not everyone is going to be 100% familiar with all the Unity callbacks. Whilst I agree all the maintainers _should_ know this is a Unity callback, I didn't want to assume that was the case. Again, happy to remove.